metriq_gym/schemas/wit.schema.json

```diff
@@ -5,25 +5,80 @@
     "description": "The WIT benchmark schema definition, describing the Wormhole-inspired teleportation benchmark configuration.",
     "type": "object",
     "properties": {
-      "benchmark_name": {
-        "type": "string",
-        "const": "WIT",
-        "description": "Name of the benchmark. Must be 'WIT' for this schema."
-      },
-      "shots": {
-        "type": "integer",
-        "description": "Number of measurement shots (repetitions) to use for the benchmark.",
-        "default": 1000,
-        "minimum": 1,
-        "examples": [1000]
-      },
-      "num_qubits": {
-        "type": "integer",
-        "description": "Number of qubits to use for the benchmark. Must be either 6 or 7.",
-        "enum": [6, 7],
-        "default": 6,
-        "examples": [6, 7]
-      }
+        "benchmark_name": {
+            "type": "string",
+            "const": "WIT",
+            "description": "Name of the benchmark. Must be 'WIT' for this schema."
+        },
+        "shots": {
+            "type": "integer",
+            "description": "Number of measurement shots (repetitions) to use for the benchmark.",
+            "default": 1000,
+            "minimum": 1,
+            "examples": [1000]
+        },
+        "n_qubits_per_side": {
+            "type": "integer",
+            "description": "Number of qubits on each side of the thermofield double state.",
+            "default": 3,
+            "minimum": 1,
+            "examples": [3, 4]
+        },
+        "message_size": {
+            "type": "integer",
+            "description": "Number of qubits used to encode the teleported message. Only 1 is currently supported for swap/transfer insertion.",
+            "default": 1,
+            "minimum": 1,
+            "examples": [1]
+        },
+        "insert_message_method": {
+            "type": "string",
+            "description": "Technique used to inject the message into the left subsystem.",
+            "enum": ["reset", "swap", "transfer"],
+            "default": "reset",
+            "examples": ["reset", "swap"]
+        },
+        "interaction_coupling_strength": {
+            "type": "number",
+            "description": "Interaction coupling strength (g) applied during the two-sided coupling step.",
+            "default": 1.5707963267948966,
```

Comment:
I am guessing those values are fractions of pi? It's odd that one can't use symbols in here. One more reason to get rid of those parameters.

Reply:
Good point. Those have since been removed (see the above comment/response).